contracts/upgradeable_contracts/native_to_erc20/ForeignBridgeNativeToErc.sol

```diff
@@ -22,7 +22,8 @@ contract ForeignBridgeNativeToErc is
         uint256 _requiredBlockConfirmations,
         uint256[] _homeDailyLimitHomeMaxPerTxArray, // [ 0 = _homeDailyLimit, 1 = _homeMaxPerTx ]
         address _owner,
-        uint256 _decimalShift
+        uint256 _decimalShift,
+        address _bridgeOnOtherSide
```

Comment:
is my understanding correct and the scenario `approve-then-relayTokens` is not supported in the bridge mode. Do you think it make sense to keep consistency and support it for all bridge modes where erc677 tokens are intended to be used? If yes, I will create a new issue.

Reply:
You are correct, `approve-then-relayTokens` was only added for the modes where `erc677` is not present in order to be able to specify an alternative receiver. I think that since `erc677` can specify a different receiver using `transferAndCall` method, `relayTokens` is not needed for those cases because it will be a duplicated functionality and from user perspective requires two transactions instead of one so I think `transferAndCall` will be always preferred

Reply:
right. but consider a following situation: a user knows that on the bridge `A` he/she can transfer tokens by pairing `approve` and `relayTokens`. Next, the same user tries to transfer tokens through the bridge `B` but fails. In both cases he/she considers the token contract as erc20 compatible. But in one case it works and does not work in another. I assumed this consistency when raised the comment initially. I understand that's it is not mandatory to have but asking your opinion whether we can suggest the user this level of usability in the future.

Reply:
Thanks for the explanation, in that case keeping consistency and allowing the users to interact in the same way with all the bridges is a nice usability feature, let's create a new issue for this.

Reply:
Ok. I will do.